oem/firstvoices/windows/src/xml/keyman_keyboardlist.xsl

```diff
@@ -275,6 +260,45 @@
                             <xsl:value-of select="copyright"/>
                         </td></tr>
                       </xsl:if>
+
+                      <!--<xsl:if test="$singlekeyboardpackage = '1' or //KeymanPackageContentKeyboardsInstalled/KeymanKeyboardInstalled[id=current()/id]">
```

Comment:
Do you prefer keeping these commented out or deleted?

Reply:
I've left them in, but commented, so that we can easily see the intentional differences between FVKeyboards and Keyman.